### PR TITLE
#52 More explicit placeholder text for search box

### PR DESCRIPTION
**Pull request by pooja2299**
Added my solution to issue #52.

### CODE REVIEW

**Comment by bryan-brancotte** [comment]
Hi @pooja2299 

Thanks for the PR, indeed the text you propose is better than what was initially there :medal_sports: . Note that adding an icon would also have been a good idea.

I can only recommend that now you grab another "good first issue".

Best wishes

**Review by hmenager — CHANGES_REQUESTED** [review]
Thank you so much @pooja2299! I just asked for two minor modifications to the code (see below)

File: index.html [hunk 1]
```diff
@@ -119,7 +119,7 @@ <h1 class="no-icon text-center">
                             type="text"
                             tabindex="1"
                             class="form-control ui-autocomplete-input search-term"
-                            placeholder="Search for concepts, type at least two characters here."
```
[the remark this answers is not in the document]
Comment (pooja2299) [reply index.html:122]: @hmenager changes done :thumbsup:. Have a look. Also @bryan-brancotte , search icon was already there so I didn't add it :blush: .

[the remark this answers is not in the document]
Comment (pooja2299) [reply index.html:122]: Also it is showing some conflict in my PR. Can you guide me to resolve it?

[the remark this answers is not in the document]
Comment (hmenager) [reply index.html:122]: ok, first let me resolve this comment, then we can discuss the conflict in PR main comments!

**Review by hmenager — APPROVED** [review]

**Comment by hmenager** [comment]
@pooja2299 to resolve the conflict here, the easiest way would be to **click on the "resolve conflict" button** on this page. It should take you to a web editor where you can see the conflict as shown in the screenshot below:
image
The conflict is shown in lines 122 to 126, with your version being in line 123 and the current main version being in line 125.
Here, all you have to do is **remove lines 122 and 124-126**, to keep the version of the code in line 123. 
Then, you can validate by clicking on the "Mark as resolved" button at the top right corner of this same page, and the conflict should be solved.

**Comment by pooja2299** [comment]
Conflict resolved!:grin: Thanks @hmenager for such detailed guidance.

**Comment by matuskalas** [comment]
Should we merge this now @bryan-brancotte & @hmenager ?

**Review by hmenager — APPROVED** [review]

**Comment by hmenager** [comment]
@bryan-brancotte will not be much available today, so since I discussed this PR yesterday with him, I won't request a formal review from him today :)

**Pull request merged**